cg/store/crud/read.py

```diff
@@ -1463,3 +1463,8 @@ def get_cases_for_sequencing_qc(self) -> list[Case]:
                 CaseFilter.HAS_SEQUENCE,
             ],
         ).all()
+
+    def get_case_ids_with_sample(self, sample_id: str) -> list[str]:
```

Comment:
```suggestion
    def get_case_ids_with_sample(self, sample_id: int) -> list[str]:
```

If you decide to go with the database id, although I would probably use the internal_id instead.

Reply:
Using the `internal_id` is not an option since the selected records are identified with the database id:s by default. Fixed the type hint in https://github.com/Clinical-Genomics/cg/pull/3507/commits/41c8b7b29beffb69bd26234e2ab22bd87627fedb